src/app/cube/home/learning-object-info/learning-outcomes/learning-outcomes.component.scss

```diff
@@ -1,15 +1,77 @@
-.content{
+@import 'vars.scss';
+
+.content {
     margin-top: 50px;
     margin-right: 50px;
+    padding-left: 0px;
     display: flex;
     flex-direction: column;
-    .taxonomy{
-      margin-bottom: 400px;
+    
+    .blooms-cards {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-evenly;
+        gap: 25px;
+        color: $light-blue;
+        padding-bottom: 300px;
+
+        .blooms-card {
+            padding: 20px 40px;
+            min-width: 175px;
+            max-width: 175px;
+            height: 100px;
+            background-color: $primary-white;
+            border: 2px solid $primary-white;
+            border-radius: 8px;
+            transform-style: preserve-3d;
+            position: relative;
+
+            .blooms-face {
+                display: block;
+                position: absolute;
+                text-align: center;
+                width: 175px;
+                backface-visibility: hidden;
+            }
+
+            .blooms-content {
+                display: flex;
+                flex-direction: column;
+                align-items: center;
+                max-height: 100px;
+                cursor: default;
+                -webkit-transform : rotateY(0deg);
+                -ms-transform     : rotateY(0deg);
+                transform         : rotateY(0deg);
+            }
+
+            .blooms-description {
+                text-align: center;
+                -webkit-transform : rotateY(-180deg);
+                -ms-transform     : rotateY(-180deg);
+                transform         : rotateY(-180deg);
+
+                p {
+                    cursor: default;
+                    margin: 0;
+                }
+            }
+
+            &:hover {
+                border: 2px solid $light-blue;
```

Comment:
I'd suggest dropping this border width down to 1px to match the rest of the borders on the homepage.